cassandra-datastax-driver/src/main/scala/com/avast/sst/datastax/DatastaxHelper.scala

```diff
@@ -1,9 +1,9 @@
 package com.avast.sst.datastax
 
-import com.datastax.oss.driver.api.core.config.{DriverOption, ProgrammaticDriverConfigLoaderBuilder => DriverBuilder}
-
-import scala.collection.JavaConverters._
-import scala.concurrent.duration.Duration
+import com.datastax.oss.driver.api.core.config.{DriverOption, ProgrammaticDriverConfigLoaderBuilder => DriverBuilder}
+
+

import scala.collection.JavaConverters._
```

Comment:
??

Reply:
problems with LF vs CRLF, I've fixed it already